.github/actions/test-khiops-install/action.yml

```diff
@@ -1,74 +1,54 @@
 ---
 name: Test Khiops Installation
-description: Tests a Khiops installation on the Iris dataset
+description: Check the outputs of khiops in the console
 runs:
   using: composite
+  # Test if the output of Khiops in the console is as expected (no mpi or java errors)
+  # We expect only one line in stdout+stderr when running 'khiops -v'
   steps:
-    # Python install: we don't use the setup-python action because of the following error:
-    # python3: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by python3)
-    - name: Python setup for Linux
-      if: runner.os == 'Linux'
+    - name: Print status for debugging
       shell: bash
+      # We force the process number to 4 because the runner has only 2 proc.
+      # With 2 proc khiops-env switch to serial and mpi is not used.
+      # The env var OMPI_MCA_rmaps_base_oversubscribe is for openmpi it corresponds to
+      # the flag --oversubscribe (Nodes are allowed to be oversubscribed)
       run: |
-        if [ -d "/etc/apt" ]
-        then
-          apt-get install -y python3 > /dev/null
-        else
-          yum install -y python3.11
-        fi
-        echo "PYTHON=python3" >> "$GITHUB_ENV"
-    - name: Python setup for Windows or macOS
-      if: runner.os == 'Windows' || runner.os == 'macOS'
-      shell: bash
-      run: echo "PYTHON=python" >> "$GITHUB_ENV"
-    - name: Set the name for the khiops laucher script
-      shell: bash
-      run: |
-        if [[ $RUNNER_OS == "Windows" ]]
-        then
-          # Put the full path in windows since bash doesn't find ".cmd" files
-          # in the path even if they are there
-          modl_path=$(which MODL)
-          khiops_bin_dir=$(dirname "$modl_path")
-          KHIOPS_SCRIPT="$khiops_bin_dir/khiops.cmd"
-          KHIOPS_CC_SCRIPT="$khiops_bin_dir/khiops_coclustering.cmd"
-          echo "KHIOPS_SCRIPT=$KHIOPS_SCRIPT" >> "$GITHUB_ENV"
-          echo "KHIOPS_CC_SCRIPT=$KHIOPS_CC_SCRIPT" >> "$GITHUB_ENV"
-        else
-          echo "KHIOPS_SCRIPT=khiops" >> "$GITHUB_ENV"
-          echo "KHIOPS_CC_SCRIPT=khiops_coclustering" >> "$GITHUB_ENV"
-        fi
-    - name: Check Khiops installation
-      shell: bash
-      run: |
-        export KHIOPS_PROC_NUMBER=4 
         export OMPI_MCA_rmaps_base_oversubscribe=true
-        "$KHIOPS_SCRIPT" -s
-        "$KHIOPS_CC_SCRIPT" -s
-    - name: Run Khiops tests
+        export KHIOPS_PROC_NUMBER=4
```

Comment:
IMHO, it's better to set these 2 env vars in an `env:` YAML section of the relevant step:

```
env:
  KHIOPS_PROC_NUMBER: 4
  OMPI_MCA_rmaps_base_oversubscribe: true
```

Reply:
I wanted to do this, but I think it is not possible for actions (only workflows)